resources/js/Pages/Public/Course/Course.vue

```diff
@@ -0,0 +1,90 @@
+<script setup>
+import PublicLayout from '@/Layouts/PublicLayout.vue'
+import BackgroundGrid from '../../../Components/BackgroundGrid.vue'
+import SectionHeader from '../../../Components/SectionHeader.vue'
+import DOMPurify from 'dompurify'
+
+defineProps({
+  course: Object,
+})
+</script>
+
+<template>
+  <PublicLayout>
+    <div class="relative isolate bg-white">
+      <BackgroundGrid />
+      <div class="py-24 sm:py-32">
+        <div class="mx-auto max-w-7xl px-6 lg:px-8">
+          <SectionHeader>
+            <template #header>
+              {{ course.name }}
+            </template>
+            <template #subheader>
+              Strona informacyjna kursu {{ course.abbreviation }}
+            </template>
+          </SectionHeader>
+
+          <div
+            class="mx-auto mt-10 border-gray-200 pt-10 sm:mt-16 sm:pt-16 md:border-t lg:mx-0 lg:max-w-none lg:grid-cols-3"
+          >
+            <h3 class="text-xl">
+              Podstawowe informacje
+            </h3>
+            <div
+              class="mx-auto mt-10 grid max-w-2xl grid-cols-1 gap-x-8 gap-y-16 uppercase lg:max-w-none lg:grid-cols-3"
+            >
+              <div class="flex-row text-center">
+                <div class="text-xl leading-8">
+                  KIERUNEK
```

Comment:
Please don't use capslocked texts. We have CSS for that. 

Plus it will be more WCAG-friendly.

Reply:
fixed